versions-maven-plugin/src/test/java/org/codehaus/mojo/versions/DisplayExtensionUpdatesMojoTest.java

```diff
@@ -57,9 +60,16 @@ public class DisplayExtensionUpdatesMojoTest {
     private DisplayExtensionUpdatesMojo mojo;
     private Path tempPath;
 
+    private PomHelper pomHelper;
+
+    private ArtifactFactory artifactFactory;
+
     @Before
-    public void setUp() throws IllegalAccessException, IOException {
-        mojo = new DisplayExtensionUpdatesMojo(mockArtifactHandlerManager(), mockAetherRepositorySystem(), null, null);
+    public void setUp() throws IllegalAccessException, IOException, MojoExecutionException {
+        openMocks(this);
```

Comment:
`@RunWith(MockitoJUnitRunner.class)`?

Reply:
Unfortunately, this is not possible, leads to this (incorrect) error message:

```
org.mockito.exceptions.misusing.UnnecessaryStubbingException: 
Unnecessary stubbings detected in test class: DisplayExtensionUpdatesMojoTest
Clean & maintainable test code requires zero unnecessary code.
Following stubbings are unnecessary (click to navigate to relevant line of code):
  1. -> at org.codehaus.mojo.versions.utils.MockUtils.mockAetherRepositorySystem(MockUtils.java:83)
```